sys.config.example

```diff
@@ -7,7 +7,7 @@
              , [ {restart_delay_seconds, 10}]
              }
            , { producers
-             , [ { <<"brod-test-topic-1">>
+             , [ { <<"brod">>
```

Comment:
used "topic" in the name to make it clear that it should be a topic name here.
if rename, perhaps add a comment behind <<"brod">> ?
e.g. <<"brod">> %% topic name